bundler/lib/bundler/source/rubygems.rb

```diff
@@ -176,7 +176,7 @@ def install(spec, options = {})
           bin_dir: bin_path.to_s,
           ignore_dependencies: true,
           wrappers: true,
-          env_shebang: true,
+          env_shebang: Bundler.settings[:env_shebang].nil? ? true : Bundler.settings[:env_shebang],
```

Comment:
I think if you add `"BUNDLE_ENV_SHEBANG" => true` to the `DEFAULT_CONFIG` constant in `Bundler::Settings`, you may be able to use `Bundler.settings[:env_shebang]` directly here?

Reply:
Thanks for the tip, much nicer indeed